src/pages/about-data.js

```diff
@@ -1,79 +1,58 @@
-                            in raw form so that you can verify any calculations and conduct your own analysis.</p>
-                            <p>OpenOUSD is an open source project which means all the code is available{' '}
-                            for the public to view and contribute to.</p>
-                        </div>
-                    </Col>
-                </Row>
-                <Row>
-                    <Col md={8}>
-                        <div>
-                            <p><strong>Data Sources</strong></p>
-                            <p>OpenOUSD raw data is posted on <a href="http://data.openoakland.org" target="_blank" rel="noopener noreferrer">
-                            data.openoakland.org</a></p>
-                            <ul>
-                                <li>
-                                    <a
-                                        href="http://data.openoakland.org/dataset/oakland-unified-school-district-ousd-expenditures-and-staffing"
-                                        target="_blank" rel="noopener noreferrer">
-                                        OUSD Expenditure and Staffing Data
-                                    </a>
-                                </li>
-                            </ul>
-                            <p><Link to="/contact/">Contact us</Link> to report an inaccuracy, offer feedback, or ask a question.</p>
-                        </div>
-                    </Col>
-                </Row>
-                <Row>
-                    <Col md={8}>
-                        <div>
-                            <p><strong>OpenOUSD Source Code</strong></p>
-                            <p>OpenOUSD is a volunteer project of <a href="https://openoakland.org" target="_blank" rel="noopener noreferrer">
-                            OpenOakland</a> and source code is avalible on GitHub.</p>
-                            <ul>
-                                <li>
-                                    <a
-                                        href="https://github.com/openoakland/openousd-site"
-                                        target="_blank" rel="noopener noreferrer">
-                                        OpenOUSD website code
-                                    </a>
-                                </li>
-                                <li>
-                                    <a
-                                        href="https://github.com/openoakland/openousd-api"
-                                        target="_blank" rel="noopener noreferrer">
-                                        OpenOUSD API code
-                                    </a>
-                                    {' '}(this API is not live, but shows how the data is queried)
-                                </li>
-                            </ul>
-                            <p><Link to="/contact/">Contact us</Link> about making contributions to the project.</p>
```

Comment:
Do you see any way to have a Gatsby page link within a Contentful rich content block? I didn't find much on that, so for now I cheated and removed links from the content. I think this is good to go, but thought I'd bring that up in case you have ideas! Hoping we don't need to build a custom renderer or something.

Reply:
Good question! I will check out Contentful and see if I can figure something out.

Reply:
I've done some Googling and I don't think this is going to be possible within Contentful -- I think we would need to build a custom renderer to support this. The easiest (and kind of hacky) approach that I can think of would be to:
- Agree on some templatized formatting to use
- Add links in Contentful in the format: `{{ROOT_URL}}/about-data`
- Implement a wrapper around `documentToReactComponents` that scans JSON and correctly replaces `{{ROOT_URL}}`
- Switch over to using that wrapper instead of using `documentToReactComponents`

Curious your thoughts on the best way to do this, and also curious on the level of priority here. Should I leave this as is -- not a link -- and keep translating more pages next week? Or is it better to focus on building out the ability to add in Gatsby links?

Reply:
Got it.  No, I don't think we need that to merge this. Can definitely do without links for now. I was just curious to get your thoughts on it since I came up empty. 

Thanks for looking into it!